Dockerfile.debian

```diff
@@ -68,6 +68,15 @@ RUN cd /tmp && \
     curl -Lo direnv https://github.com/direnv/direnv/releases/download/v2.15.2/direnv.linux-amd64 && \
     chmod +x direnv && mv direnv /usr/local/bin
 
+# docker
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add - && \
+   add-apt-repository \
+     "deb [arch=amd64] https://download.docker.com/linux/debian \
+     $(lsb_release -cs) \
+     stable" && \
+   apt-get -y update && \
+   apt-get -y install docker-ce
```

Comment:
SHOULD: add `apt-get clean` and maybe `apt-get autoremove` here and after apt ops above to minimize image size.

Reply:
Done